iDiff/fileDiff_test_processor.py

```diff
@@ -0,0 +1,35 @@
+import json
+import re
+import sys
+
+
+def _process_test_diff(file_path):
+    f = open(file_path)
+    diff = json.load(f)
+    f.close()
+
+    diff["Adds"] = _trim_file_names(diff["Adds"])
+    diff["Dels"] = _trim_file_names(diff["Dels"])
+
+    f = open(file_path, 'w')
+    json.dump(diff, f, indent=4)
+
+
+def _trim_file_names(files):
+    trimmed_files = []
+    for file in files:
+        trimmed_file = _trim_layer_hash(file)
+        trimmed_files.append(trimmed_file)
+    return sorted(trimmed_files)
+
+
+def _trim_layer_hash(filename):
+    hash_match = re.match(r'^([a-z|0-9]{64})/', filename)
+    if hash_match:
+        hash = hash_match.group(1)
+        return filename[len(hash):]
```

Comment:
can you use `re.sub()` here instead of doing string indexing? it's a bit cleaner.